code/modules/jobs/job_types/station_trait/human_ai.dm

```diff
@@ -0,0 +1,170 @@
+		if(!inactive_core.is_available())
+			continue
+		GLOB.latejoin_ai_cores -= inactive_core
+		inactive_core.available = FALSE
+		. = inactive_core.loc
+		qdel(inactive_core)
+		return
+	var/list/primary_spawn_points = list() // Ideal locations.
+	var/list/secondary_spawn_points = list() // Fallback locations.
+	for(var/obj/effect/landmark/start/ai/spawn_point in GLOB.landmarks_list)
+		if(spawn_point.used)
+			secondary_spawn_points += list(spawn_point)
+			continue
+		if(spawn_point.primary_ai)
+			primary_spawn_points = list(spawn_point)
+			break // Bingo.
+		primary_spawn_points += spawn_point
+	var/obj/effect/landmark/start/ai/chosen_spawn_point
+	if(length(primary_spawn_points))
+		chosen_spawn_point = pick(primary_spawn_points)
+	else if(length(secondary_spawn_points))
+		chosen_spawn_point = pick(secondary_spawn_points)
+	else
+		CRASH("Failed to find any AI spawn points.")
+	chosen_spawn_point.used = TRUE
+	return chosen_spawn_point
+
+/datum/job/human_ai/special_check_latejoin(client/latejoin_client)
+	for(var/obj/structure/ai_core/latejoin_inactive/latejoin_core as anything in GLOB.latejoin_ai_cores)
+		if(latejoin_core.is_available())
+			return TRUE
+	return FALSE
+
+/datum/job/human_ai/announce_job(mob/living/joining_mob)
+	. = ..()
+	if(SSticker.HasRoundStarted())
+		minor_announce("Due to a research mishaps, [joining_mob] has been sent to be your replacement AI at [AREACOORD(joining_mob)]. Please treat them with respect.")
+
+/datum/job/human_ai/get_radio_information()
+	return "<b>Prefix your message with :b to speak with cyborgs.</b>"
+
+/datum/outfit/job/human_ai
+	name = "Human AI"
+	jobtype = /datum/job/human_ai
+
+	id = /obj/item/card/id/advanced/robotic
+	id_trim = /datum/id_trim/job/human_ai
+	backpack_contents = list(
+		/obj/item/door_remote/omni = 1,
+		/obj/item/machine_remote = 1,
+		/obj/item/secure_camera_console_pod = 1,
+	)
+	implants = list(
+		/obj/item/implant/teleport_blocker,
+	)
+
+	uniform = /obj/item/clothing/under/rank/station_trait/human_ai
+	belt = /obj/item/modular_computer/pda/human_ai
+	ears = /obj/item/radio/headset/silicon/human_ai
+	glasses = /obj/item/clothing/glasses/hud/diagnostic
+
+	suit = /obj/item/clothing/suit/costume/cardborg
+	head = /obj/item/clothing/head/costume/cardborg
+
+	l_pocket = /obj/item/laser_pointer/infinite_range //to punish borgs, this works through the camera console.
+	r_pocket = /obj/item/assembly/flash/handheld
+
+	l_hand = /obj/item/paper/default_lawset_list
+
+/datum/outfit/job/human_ai/post_equip(mob/living/carbon/human/equipped, visualsOnly)
+	. = ..()
+	if(visualsOnly)
+		return
+	if(!equipped.get_quirk(/datum/quirk/body_purist))
+		var/obj/item/organ/internal/tongue/robot/cybernetic = new()
+		cybernetic.Insert(equipped, special = TRUE, movement_flags = DELETE_IF_REPLACED)
+		//you only get respect if you go all the way, man.
+		ADD_TRAIT(equipped, TRAIT_COMMISSIONED, INNATE_TRAIT)
+	equipped.faction += list(FACTION_SILICON, FACTION_TURRET)
```

Comment:
```suggestion
	equipped.faction |= list(FACTION_SILICON, FACTION_TURRET)
```

Reply:
There's no real difference between the two, I just prefer this